src/Properties/ClientOptions.php

```diff
@@ -4,27 +4,16 @@
 
 class ClientOptions
 {
-    public string $endpoint;
-
-    public function setEndpoint(string $string)
-    {
-        $this->endpoint = $string;
-        return $this;
-    }
-
-    public bool $useBundle;
-
-    public function setUseBundle(bool $use)
-    {
-        $this->useBundle = $use;
-        return $this;
-    }
-
-    public string $apiKey;
-
-    public function setApiKey(string $key)
+    /**
+     * @param string $endpoint
+     * @param bool $useBundle
+     * @param string $apiKey
+     */
+    public function __construct(
```

Comment:
Great simplification :+1: 
You might delete the @param annotations since the data is already in the method signature.

Reply:
With `@param` PHPStorm displays hints little in a different way, which is more useful.

Is it only for me works?)